code/hsec-tools/test/golden/EXAMPLE_ADVISORY.md.golden

```diff
@@ -42,8 +42,8 @@ Right
                         , affectedVersionRangeFixed = Nothing
                         }
                     ]
-                , affectedArchitectures = Nothing
-                , affectedOS = Nothing
+                , affectedArchitectures = Just []
+                , affectedOS = Just []
                 , affectedDeclarations = []
```

Comment:
what is the justification for this golden change.

Reply:
I think it's actually non-sense, the parse should return `Nothing` here. the fields do not exist.